protocols/newFile/handlers.go

```diff
@@ -0,0 +1,241 @@
+package newFile
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"io"
+
+	"github.com/functionland/go-fula/drive"
+	fxiface "github.com/functionland/go-fula/fxfs/core/iface"
+	files "github.com/ipfs/go-ipfs-files"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/libp2p/go-libp2p-core/network"
+)
+
+// Root handler for newFile protocol
+// It reads 2 bytes from the stream which determines the size of the request messeage
+// It then reads the request message and passes the stream to the appropriate handler
+func Handle(ctx context.Context, api fxiface.CoreAPI, ds drive.DriveStore, s network.Stream) error {
+	fmt.Println("handling")
```

Comment:
Use logging library instead. Search for similar lines across the PR and replace as needed